config/samples/numaplane.numaproj.io_v1alpha1_isbservicerollout.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   strategy:
     progressive:
-      assessmentSchedule: "0,60,10"
+      assessmentSchedule: "0,480,60,10"
```

Comment:
Actually, are we using the 480 or the 60? If not, can we just set those both to 0 for now? (I realize this was already an issue in the existing code)